web_hide_db_manager_link/README.rst

```diff
@@ -1,37 +1,57 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
+=============================================
 Hide link to database manager in login screen
 =============================================
 
 This module hides the "Manage Databases" link at the bottom of login screen.
 
-The image in *web_hide_db_manager_link/static/src/img/screen.png* shows the resulting loging screen.
-
-.. image:: web_hide_db_manager_link/static/src/img/screenshot.png
 
 Installation
 ============
 
 To install this module, you need to:
 
  * Go to Settings / Local Modules
- * Search by module name "Hide link to database manager in login screen" or by module technical name "*web_hide_db_manager_link*" 
+ * Search by module name "Hide link to database manager in login screen" or
+   by module technical name "*web_hide_db_manager_link*"
  * Click install button
 
+Usage
+=====
+
+When this module is installed the link "Manage Databases" will be hidden in
+the login screen.
+
+The image in *web_hide_db_manager_link/static/src/img/screen.png* shows the
+resulting loging screen.
+
+.. image:: web_hide_db_manager_link/static/src/img/screenshot.png
+
+In order to access to the database management, the administrator should use a
+specific URL <myserver>/web/database/manager
+
+.. image:: https://odoo-community.org/website/image/ir
+.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/162/9.0
+
 Known issues
 ============
 
-As Odoo v8 templates only live in the database, this module must be installed in each database you want this to be in effect.
+* As Odoo v8 templates only live in the database, this module must be
```

Comment:
@jbeficent is this still the case in v9?

I'm surprised that such a module (that is by nature database independent) needs to be installed in databases.